iree/compiler/Dialect/HAL/Target/LLVM/LLVMIRPasses.cpp

```diff
@@ -100,8 +100,8 @@ LogicalResult runLLVMIRPasses(const LLVMTargetOptions &options,
                 compileKernel, recover, moduleUseAfterScope, useOdrIndicator));
             modulePassManager.addPass(
                 createModuleToFunctionPassAdaptor(llvm::AddressSanitizerPass(
-                    {compileKernel, recover, useAfterScope,
-                     llvm::AsanDetectStackUseAfterReturnMode::Runtime})));
+                    compileKernel, recover, useAfterScope,
+                    llvm::AsanDetectStackUseAfterReturnMode::Runtime)));
```

Comment:
Pretty sure this was just creating a struct? The method signature changed in https://github.com/llvm/llvm-project/commit/36d51386195e3d606e0d40495f1135ab180bd6ae to take a struct. Did you update your version of LLVM?